### ISSUE TITLE
Hi there! 

### ISSUE THREAD

**Issue by flatoy** [blake-regalia/node-pj#1]
I'm working on a CLI that makes it easier to work with node modules. It works directly with the package.json file so I'd love to call the tool `pj` but I noticed you have this package that's been dead for 8-9 years. 

Any chance you'd be willing to transfer this package name to me? 🙏 

The tool I'm working on will be free and open sourced.

**Comment by blake-regalia** [comment]
Absolutely. Please give me your npm username so i can invite you as a maintainer.

**Comment by flatoy** [comment]
awesome! It's `vhflat`

**Comment by flatoy** [comment]
@blake-regalia ping ☝️
